archive/unix_pack.ksy

```diff
@@ -0,0 +1,107 @@
+meta:
+  id: unix_pack
+  title: Unix pack(1)-compressed data
```

Comment:
is `1` a man section? If it is, I don't think it is a good idea to put it into name of a program because it is not a part of it.

Reply:
Correct, this is how man pages usually refer to command-line tools. It seems that this is done less often outside man pages (and even in man pages the section number is often only included the first time a tool is mentioned, not every time), so I'll probably remove the section numbers in most places.